src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  if (days >= 3 && days < 7) {
```

Comment:
Consider using constants for magic numbers like 3, 7, 40, 20, and 50. This will make the code more readable and maintainable. 🤓